### PR TITLE
Switch from colors to chalk

### PR DESCRIPTION
**Pull request by jason-fox**
@fgalan @AlvaroVega - related #62 - you probably want to drop `colors` altogether and user `chalk` instead.

```

  99 passing (395ms)

---------------|----------|----------|----------|----------|-------------------|
File           |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
---------------|----------|----------|----------|----------|-------------------|
All files      |    99.09 |    98.11 |      100 |    99.09 |                   |
 formatters.js |    98.41 |    96.97 |      100 |    98.41 |               195 |
 logops.js     |      100 |      100 |      100 |      100 |                   |
---------------|----------|----------|----------|----------|-------------------|

```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,15 +37,15 @@
     "sinon-chai": "^3.3.0",
     "tslint": "^5.18.0",
     "tslint-config-typings": "^0.3.1",
-    "typescript": "^2.0.3"
+    "typescript": "^4.5.4"
```
Comment (jason-fox) [inline package.json:40]: This dependency update was required to get `npm run test:ts"` to pass since it is now ES2017 not ES2015

**Review by jmendiara — APPROVED** [review]
Thanks a lot! 🔝 

I prefer this approach to #62, to avoid future problems with this dependency.

**Comment by jmendiara** [comment]
Released as [2.1.2](https://github.com/telefonicaid/logops/releases/tag/2.1.2)
Published as [2.1.2](https://www.npmjs.com/package/logops/v/2.1.2)

**Comment by jason-fox** [comment]
Someone should run the following query:

https://github.com/search?q=org%3Atelefonicaid+%22logops%22%3A++extension%3Ajson&type=Code

and update the logops dependency across all 22 repos to 2.1.2 - at the moment a selection have been completed.

**Comment by fgalan** [comment]
Thanks for the query. Another useful ones:

* https://github.com/search?q=org%3Atelefonicaid+%22colors%22%3A++extension%3Ajson&type=Code
* https://github.com/search?q=org%3Atelefonicaid+%22faker%22%3A++extension%3Ajson&type=Code

**Pull request merged**